### DISCUSSION TITLE
Docker datasource support for stabilityDays/releaseTimestamp

### DISCUSSION THREAD

**Discussion by smithc14** [renovatebot/renovate#20049] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I am hoping to use stabilityDays with docker digest pinning to auto merge an update to a FROM reference when it has existed for a certain period of time.

I believe I have read that stabilityDays works when renovate has a releaseTimestamp field in the `updates` object under "packageFiles with updates" in the log. Is this correct?

If that assumption is correct then I'm wondering if any docker datasource supports finding a releaseTimestamp? 

I know my images have the  "org.opencontainers.image.created" label in their metadata...

### What you tried so far.

I have used stabilityDays to no effect in my renovate config and I have found no releaseTimestamp in the updates objects for my docker images.

### Relevant debug logs

<details><summary>Logs</summary>

```
{
            "depName": "ghcr.io/macewanu/peoplesoft-base",
            "currentValue": "oel7",
            "currentDigest": "sha256:e91bd22f0ee2e3dd874052c739a93f68de2bd0922360989ed58ec816e75fdee7",
            "replaceString": "ghcr.io/macewanu/peoplesoft-base:oel7@sha256:e91bd22f0ee2e3dd874052c739a93f68de2bd0922360989ed58ec816e75fdee7",
            "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
            "datasource": "docker",
            "depType": "final",
            "depIndex": 1,
            "updates": [
              {
                "updateType": "digest",
                "newValue": "oel7",
                "newDigest": "sha256:f2b6ba2aa56984c0f13eeab6c062a5699304b372358f35d19a673b51b2be9c3c",
                "branchName": "renovate/main-ghcr.io-macewanu-peoplesoft-base-oel7"
              }
            ],
            "warnings": [],
            "versioning": "docker"
          }
```

</details>

**Comment by viceice** [comment, marked as answer]
not possible, as we would need to fetch every docker tag to read that value. and we don't want to do that

**Reply by smithc14** [reply]
So it's definitely not supported with docker. I'll find another way to achieve what I'm hoping for. Thanks for the great tool.
